torch/csrc/distributed/c10d/comm.cpp

```diff
@@ -32,7 +32,13 @@ class BroadcastWork {
         flat_tensor_.front(), bucket_tensors_);
     TORCH_INTERNAL_ASSERT(output_tensors.size() == bucket_tensors_.size());
     for (const auto i : c10::irange(output_tensors.size())) {
-      bucket_tensors_[i].copy_(output_tensors[i], /*non_blocking=*/true);
+      // if output_tensor is empty, no need to copy it back,
```

Comment:
Is this accidentally included from the other BE PR?

Reply:
yes, it is the change this PR stacks on...